userguide/content/en/docs/adding-content/diagrams-and-formulae/index.md

```diff
@@ -35,6 +35,13 @@ $$\tag*{(1)} P(E) = {n \choose k} p^k (1-p)^{n-k}$$
 The probability of getting \\(k\\) heads when flipping \\(n\\) coins is:
 $$\tag*{(1)}  P(E) = {n \choose k} p^k (1-p)^{n-k}$$
 
+You can also use [GLFM's math blocks](https://docs.gitlab.com/ee/user/markdown.html#math):
```

Comment:
You are intoducing a new formula (e = m * c2). I think it will be clearer if we refer to the expression given and rendered before: 

`````
The probability of getting \\(k\\) heads when flipping \\(n\\) coins is:
$$
\tag*{(1)}  P(E) = {n \choose k} p^k (1-p)^{n-k}
$$

You can also use [GLFM's math blocks](https://docs.gitlab.com/ee/user/markdown.html#math) to render this expression:
````markdown
```math
\tag*{(1)} P(E) = {n \choose k} p^k (1-p)^{n-k}
```
````
`````

Reply:
Good point, fixed.